Source/Data/Collections/MessageCollection.swift

```diff
@@ -1,39 +1,25 @@
-import ObjectMapper
+import Foundation
 
-internal class MessageCollection: Mappable {
+internal struct MessageCollection: Codable, Equatable {
```

Comment:
Because we do not use `Mappable` anymore where possible I've converted the `classes` to `structs`.